README.md

```diff
@@ -1,11 +1,24 @@
 # vmpooler-deployment
 
+- [vmpooler-deployment](#vmpooler-deployment)
+  - [Docker Registry](#docker-registry)
+  - [Helm Repository](#helm-repository)
+    - [Adding / updating charts](#adding--updating-charts)
+  - [Development](#development)
+    - [Docker Compose URLs](#docker-compose-urls)
+    - [Deploy Chart for Testing](#deploy-chart-for-testing)
+  - [Releasing](#releasing)
+  - [Contributing](#contributing)
+  - [License](#license)
+
 This repo contains Dockerfiles and a Helm chart that can be used to deploy [VMPooler](https://github.com/puppetlabs/vmpooler). The Release Engineering team at Puppet uses the code here as part of operating our VMPooler instances.
 
 ## Docker Registry
 
 The GitHub Actions in this repository publish images to GitHub Packages. You can browse the VMPooler containers [here](https://github.com/puppetlabs/vmpooler-deployment/pkgs/container/vmpooler-deployment%2Fvmpooler).
 
+The vmpooler docker image is tagged using 
```

Comment:
I think you meant to say more here. Just wanted to let you know if it was an accident.

Reply:
Ah thanks, definitely saw a squirrel or something in the middle of that sentence.